test/cli/migrate-make.spec.js

```diff
@@ -1,11 +1,57 @@
 'use strict';
 
+const fs = require('fs');
 const path = require('path');
 const { FileTestHelper, execCommand } = require('cli-testlab');
 const expect = require('chai').expect;
 
 const KNEX = path.normalize(__dirname + '/../../bin/cli.js');
 
+// start: utils for --stub option tests //
+const migrationStubSetup = (fileHelper) => {
+  fileHelper.registerGlobForCleanup(
+    'test/jake-util/knexfile_migrations/*_somename.js'
+  );
+  fileHelper.createFile(
+    process.cwd() + '/knexfile.js',
+    `
+      module.exports = {
+        development: {
+          client: 'sqlite3',
+          connection: {
+            filename: __dirname + '/test/jake-util/test.sqlite3',
+          },
+          migrations: {
+            directory: __dirname + '/test/jake-util/knexfile_migrations',
+          },
+        }
+      };
+      `,
+    { isPathAbsolute: true }
+  );
+};
+
+const getMigrationFileContents = (fileHelper) => {
+  const [migrationFileName] = fs
```

Comment:
You don't need this. As I've said, there's a new method in cli-testlab 1.7.0 (just bump the version) called `getFileGlobTextContent` specifically for this use-case.